demo/jfx/src/main/java/com/acrolinx/client/sidebar/demo/jfx/AcrolinxDemoClientJFX.java

```diff
@@ -89,17 +77,4 @@ private TextArea getTextArea()
         textArea.setPrefWidth(600);
         return textArea;
     }
-
-    private static Stage getPopUpStage(final String url)
```

Comment:
Why was this code removed?

Reply:
because the method was never used (= dead code)